templates/categories.tpl

```diff
@@ -1,5 +1,5 @@
 <ul class="categories" itemscope itemtype="http://www.schema.org/ItemList">
-	<p>Categories</p>
+	<p class="cat-title">[[category:cat_title:Categories]]</p>
```

Comment:
Should be written `[[header.categories]]`, possible. I'd like @barisusakli's comments on where the exact language key should be placed